platform/migration/platform-migratable-util/src/main/java/org/codice/ddf/migration/util/MigratableUtil.java

```diff
@@ -175,7 +176,9 @@ private String getSystemProperty(String property) throws MigrationException {
         String prop = System.getProperty(property);
 
         if (StringUtils.isBlank(prop)) {
-            String message = String.format("System property %s is not set.", property);
+            String message = String.format(
+                    "Failed to export configurations: System property [%s] is not set.",
```

Comment:
Is it worth pulling the first part out to a common area since it's being repeated in all the messages?